github_pr.py

```diff
@@ -149,6 +149,47 @@ def _return_specific_comment_prs(all_prs, filters):
         return []
 
 
+def _check_approved_mergers(approved_users, comment_users):
+    comment_approved_users = [comment_user for comment_user in comment_users if comment_user in approved_users]
+    if not comment_approved_users:
+        raise AssertionError("No approved mergers were found in comments - Approved mergers: %s" % approved_users)
```

Comment:
You should define your own exception. This is because one of the underlying system libraries could conceivably raise the same exception as you and you could accidentally catch theirs instead of yours and this could cause unexpected behavior.

```python
class NoApproversError(Exception):
    '''raise this when there's a lookup error for my app'''

raise NoApproversError("No approved mergers were found in comments - Approved mergers: %s" % approved_users)
```